src/app/games/apagos/ApagosDummyMinimax.ts

```diff
@@ -13,6 +14,10 @@ export class ApagosDummyMinimax extends PlayerMetricsMinimax<ApagosMove, ApagosS
         }
         return ApagosMove.ALL_MOVES.filter(isLegal);
     }
+}
+
+export class ApagosDummyHeuristic extends PlayerMetricHeuristic<ApagosMove, ApagosState> {
```

Comment:
not really dummy, it has the smartness to do like it's already the end of the game :p